src/prpy/tsr/tsr.py

```diff
@@ -41,6 +48,8 @@ def __init__(self, T0_w=None, Tw_e=None, Bw=None,
             Tw_e = numpy.eye(4)
         if Bw is None:
             Bw = numpy.zeros((6, 2))
+        elif any((Bw[:, 1]-Bw[:, 0]) > 2*pi + EPSILON):
+            raise(ValueError('Bw range must be within 2*PI'))
```

Comment:
@psigen suggested a simple and elegant solution [which @mkoval might also have alluded to in his earlier post]: treat the bounds as signed, i.e. as `from -> to` instead of `[min, max]` and modify `Bw[i,1]` to `Bw[i,1] + 2pi` if `Bw[i,1] < Bw[i,0]`. This will shift it out of the `pi` wraparound. Of course, this means, `Bw[i,1]` will no longer be bounded `[-pi,pi]` but can be as high as `pi+2pi=3pi` for example if we want the almost great circle `[pi, pi-eps]`. But I'm willing to live with that.